src/coreclr/jit/lowerxarch.cpp

```diff
@@ -6321,8 +6321,8 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
                                     MakeSrcContained(node, op1);
                                     // MultiplyNoFlags is a Commutative operation, so swap the first two operands here
                                     // to make the containment checks in codegen significantly simpler
-                                    *(originalArgList->pCurrent())         = op2;
-                                    *(originalArgList->Rest()->pCurrent()) = op1;
+                                    originalArgList->Current()         = op2;
```

Comment:
This is now calling [`GenTreeArgList::Current`](https://github.com/dotnet/runtime/blob/f53ae479d08d2eaee7f02b335a872a4be0287550/src/coreclr/jit/gentree.h#L3626), which is identical in implementation.